extensions/amp-auto-ads/amp-auto-ads.md

```diff
@@ -110,6 +110,22 @@ fetched from a 3rd party ad network at the URL defined in
     <td></td>
     <td>The potential places where ads can be inserted on the page.</td>
   </tr>
+  <tr>
+    <td>attributes</td>
+    <td>No</td>
+    <td>Object&lt;string, string&gt;</td>
+    <td>{}</td>
+    <td>
+      A map from attribute name to value for attrbiutes to apply to all
+      <code>&lt;amp-ad&gt;</code>s injected using this configuration. Only the
```

Comment:
Instead of `<code>&lt;amp-ad&gt;</code>s` how about `<code>&lt;amp-ad&gt;</code> elements`